### PR TITLE
chore: first stab at documenting SLOs

### PR DESCRIPTION
**Pull request by SiTaggart**
Adding docs around our SLIs and SLOs for our Front End services.

## Changes in this PR

Adding two pages of documentation detailing the SLIs, SLOs and monitors we set up for our Paste Website and Remix, loosely following this example https://wiki.hq.twilio.com/display/PRnR/Example+SLO+Document

### CODE REVIEW

File: internal-docs/engineering/sli-slo/doc-site.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| Availability | Proportion of successful searches                        | 99.99%  | [Monitor](https://app.datadoghq.com/monitors/74642104) |
+| Availability | Proportion of successful roadmap views                   | 99.99%  | Monitor                                                |
+| Performance  | Page load time under industry standard                   | < 3s    | [Monitor](https://app.datadoghq.com/monitors/74354008) |
+| Performance  | Web vital - Avg Largest Contentful Paint                 | < 4s    | [Monitor](https://app.datadoghq.com/monitors/74352039) |
+| Performance  | Web vital - Tokens list Largest Contentful Paint         | < 4s    | [Monitor](https://app.datadoghq.com/monitors/74354388) |
+| Performance  | Web vital - Avg First Input Delay                        | < 300ms | [Monitor](https://app.datadoghq.com/monitors/74353421) |
+
+## Rationale
+
+Pure Front End SLOs are a uncommon. We prioritize availability base on peoples success of "buidling" with Paste, and web performance to exemplify good performance best practices.
```
Comment (TheSisb) [inline internal-docs/engineering/sli-slo/doc-site.md:40]:
```suggestion
Pure Front End SLOs are a uncommon. We prioritize availability based on peoples success of "buidling" with Paste, and web performance to exemplify good performance best practices.
```

**Review by TheSisb — APPROVED** [review]

File: internal-docs/engineering/sli-slo/doc-site.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+| -------- | ------- | ------- | ------------------------------------------------------------------- |
+| External | Netlify | 99.999% | If Netlify breaches SLA the whole site will go down, with no backup |
+
+## SLIs and SLOs
+
+| Category     | SLI                                                      | SLO     | Notes                                                  |
+| ------------ | -------------------------------------------------------- | ------- | ------------------------------------------------------ |
+| Availability | Proportion of successful requests to the Paste Docs site | 99.99%  | [Monitor](https://app.datadoghq.com/monitors/74640746) |
+| Availability | Proportion of successful searches                        | 99.99%  | [Monitor](https://app.datadoghq.com/monitors/74642104) |
+| Availability | Proportion of successful roadmap views                   | 99.99%  | Monitor                                                |
```
Comment (shleewhite) [inline internal-docs/engineering/sli-slo/doc-site.md:32]: Question: is this monitor supposed to be a link?

Comment (SiTaggart) [reply internal-docs/engineering/sli-slo/doc-site.md:32]: Not yet, not all of them have been built yet. We don't actually need them to be running ot get sign off, but I thought I'd build a few to get the ball rolling and have some prior art for folks to copy.

File: internal-docs/engineering/sli-slo/doc-site.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+| Performance  | Web vital - Tokens list Largest Contentful Paint         | < 4s    | [Monitor](https://app.datadoghq.com/monitors/74354388) |
+| Performance  | Web vital - Avg First Input Delay                        | < 300ms | [Monitor](https://app.datadoghq.com/monitors/74353421) |
+
+## Rationale
+
+Pure Front End SLOs are a uncommon. We prioritize availability based on peoples success of "buidling" with Paste, and web performance to exemplify good performance best practices.
+
+## Error Budget
+
+Error budgets are only defined per availability monitor.
```
[the remark this answers is not in the document]
Comment (SiTaggart) [reply internal-docs/engineering/sli-slo/doc-site.md:44]: Ah, it means that error budgets only exist for the availability monitors. If that helps, I can reword it to be that.

File: internal-docs/engineering/sli-slo/readme.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# SLIs, SLOs and Monitors
```
Comment (zahnster) [inline internal-docs/engineering/sli-slo/readme.md:1]: Question: Is this intended to be fleshed out in this PR? If not, should we just remove it for now?

Comment (SiTaggart) [reply internal-docs/engineering/sli-slo/readme.md:1]: This particular page? Yeah, can be removed. I'm not against placeholders as interim things either

File: internal-docs/engineering/sli-slo/doc-site.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+[Dashboard](https://app.datadoghq.com/dashboard/p79-36e-cxi/paste-monitoring-dashboard?from_ts=1656022950408&to_ts=1656026550408&live=true)
+
+## Critical User Journeys
+
+1. The static website that serves guidelines and documentation for components, enables product designers and engineers internally and externally, to understand how to use Paste to build Twilio products and build on the Twilio Platform with.
+2. The static site serves a search functionality to enable users to find documentation quicker.
+3. The static site houses a list of design tokens which codifies the Twilio Design language, and provides a search / filter functionality.
+4. The static site houses the Paste roadmap to inform consumers of what is coming next.
+
+## Golden Signals Metrics
```
Comment (nkrantz) [inline internal-docs/engineering/sli-slo/doc-site.md:14]: suggestion (non-blocking): remove heading until content is ready?

**Review by zahnster — APPROVED** [review]

File: internal-docs/engineering/sli-slo/doc-site.md (hunk 1 above)
Comment (nkrantz) [inline internal-docs/engineering/sli-slo/doc-site.md:40]:
```suggestion
Pure Front End SLOs are uncommon. We prioritize availability based on people's success of "building" with Paste, and web performance to exemplify good performance best practices.
```

**Review by nkrantz — APPROVED** [review]
just some non-blocking nitpicky comments (non-blocking since they're internal)

File: internal-docs/engineering/sli-slo/doc-site.md (hunk 5 above)
Comment (nkrantz) [reply internal-docs/engineering/sli-slo/doc-site.md:14]: ah
